Clash - Arcane Artillery (#115)
qwrtln/Homm3BG-mission-book: draft-scenarios/clash/arcane_artillery.tex

% !TeX spellcheck = en_US
\addscenariosection{1}{Clash Scenario}{Arcane Artillery}{\images/precision.png}

\begin{multicols}{2}

\textbf{Author:} NeuroN

\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1279029213839626313}{Archon Studios Discord}

\textit{The ancient ruins are filled with arcane magic and technology, we must take control of them before our enemies can.

> Some pieces of arcane artillery are scattered aRound the ruins, they can help us kill the dragons resting in them.}

Reviewer: ```suggestion
Some pieces of arcane artillery are scattered around the ruins, they can help us kill the dragons resting in them.}
```

\textbf{Starting Units:}
\begin{itemize}
  \item a Few of each \svgunit{bronze} Unit
\end{itemize}

\textbf{Town Buildings:} \svgunit{bronze} Dwelling

\textbf{Map Tile Pool:} Each player takes 1 random Far (II--III) Map Tile, it must contain a Settlement

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles and arrange them as shown in the Scenario map layout:

\begin{itemize}
  \item If playing Alliance mode, alternate starting Tile placement such that allies do not start adjacent to each other.
\end{itemize}

\textbf{For a 2-player Scenario:}
\begin{itemize}
  \item 2 × Starting (I) Map Tile
  \item 4 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
\end{itemize}

\textbf{For a 3-6 player Scenario:}
\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
\end{itemize}

\subsection*{\MakeUppercase{Victory Conditions}}

> To win the Scenario, a Hero must Flag the Dragon Utopia Field.

Reviewer 2: ```suggestion
First Player to flag Dragon Utopia wins the Scenario.
```
Phrased to be easier to read.

\subsection*{\MakeUppercase{Defeat Conditions}}

> At the end of the \nth{12} Round, if there is no winner, all players lose the Scenario.

Reviewer 2: ```suggestion
At the end of the \nth{12} Round, if the Dragon Utopia is not flagged by any player, all players lose the Scenario.
```
Lose condition now clearly states required game state.

\subsection*{\MakeUppercase{Timed Events}}

\textbf{\nth{6} Round:}
\begin{itemize}

>   \item Remove all Black cubes from every Tile.

Reviewer: ```suggestion
  \item Remove all Black Cubes from every Tile.
```

\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}

During this Scenario:

\begin{itemize}
  \item Heroes cannot enter starting Tiles other than their own.

>   \item A Player may not use the Diplomacy card to skip Combat on the Dragon Utopia Field.
>   \item Arcane Spikes are represented by cubes not used by any player's Faction.

Reviewer: ```suggestion
  \item A Player may not use the Diplomacy Card to skip Combat on the Dragon Utopia Field.
  \item Arcane Spikes are represented by Cubes not used by any player's Faction.
```

>   \item Arcane Spikes are represented by cubes not used by any player's Faction.

Reviewer 2: I suggest to move "TIMED EVENTS" and "ADDITIONAL RULES" sections entirely to page 2:
![image](https://github.com/user-attachments/assets/7956d18d-d015-4d13-8301-df7cdbcdffee)

This way, the information relevant to the player during the match is entirely on one page.
Page 1 - all info needed for set up, before the scenario actualy starts
Page 2 - all info needed only during the actual scenario

This should improve UX for a reader.
The negative space left can be filled with some graphics:
![image](https://github.com/user-attachments/assets/506bad94-06f0-4f0a-b141-bbf6a482c253)

  \item Obelisks may be Flagged. 

>   \item Each Obelisk can only be Flagged by one Player at a time.

Reviewer: @xNeuroNx Aren't they flaggable by default? If so, these bullet points should be combined.

Reviewer 2: They are flaggable. The first bulletpoint is redundant and should be removed.

>   \item When first Flagged, that Player gains one Arcane Spike.

Reviewer 2: ```suggestion
  \item Whenever an Obelisk is visited for the first time, the visiting player gains one Arcane Spike.
```
Otherwise this rule does not specify what is being flagged. Also, this way it's consistant with the official scenario book's wording.

>   \item At the start of each Player's turn, that Player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of captured Obelisks they have.

Reviewer 2: "Captured" is not a official key word. "Flagged"/"Controlled" follows offiicial convention.
```suggestion
  \item At the start of each Player's turn, that Player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of Obelisks controlled by them.
````

\end{itemize}

> \vspace{1em}
> 
> \includegraphics[width=\linewidth, keepaspectratio]{\art/prayer.png}
> 
> \vspace*{\fill}

Reviewer: ```suggestion
\vspace*{\fill}

\includegraphics[width=\linewidth, keepaspectratio]{\art/prayer.png}

\vspace*{\fill}
```

\columnbreak

\textbf{Players can spend one Arcane Spike during combat to cast:}

\hommtable[]{15}{
  \begin{tabularx}{\linewidth}{X}
    \darkcell{\raisebox{-.2\height}{} \textbf{Arcane Artillery}}\\
    \lightcell[1.8]{\raisebox{-.25\height}{\svg{activation_white}} Deal 1 \svg{damage-table} to target Unit by placing an Arcane Spike on it.} \\ 
    \lightcell[3.8]{\raisebox{-.25\height}{\svg{ongoing_white}} A Unit with an Arcane Spike placed on it has its special ability negated \\ and receives +1 \svg{damage-table} from every source of \\ \svg{damage-table} and attacks for every Arcane Spike placed on them (even if they were to receive 0 damage)} \\
  \end{tabularx}
}

\textbf{Additionally:}

\begin{itemize}

>   \item Arcane Artillery ignores Azure Dragon's Special Ability when cast during \textbf{Level VII} combat.
>   \item Arcane Artillery does not count toward your combat Round Spell limit.
>   \item Arcane Artillery is considered to be a Spell of every school of magic of \svg{empower} 0.

Reviewer 2: ```suggestion
  \item Arcane Artillery is considered to be a Spell of every school of magic of \svg{empower} 0.
  \item Arcane Artillery does not count toward your combat Round Spell limit.
  \item Arcane Artillery ignores Azure Dragon's Special Ability when cast during \textbf{Level VII} combat.
```
These rules should be swapped around, since the 2nd rule refers to arcane artillery being a spell before the rules establish that is being considered a spell.
![image](https://github.com/user-attachments/assets/82bcea79-b2d4-40e6-b65d-66340533a0a9)

>   \item Players can cast Srcane Srtillery during combat involving opponent's Heroes \textbf{during the neutral Unit's activation}.

Reviewer 2: ```suggestion
  \item Players can cast Arcane Artillery during combat involving opponent's Heroes \textbf{during the neutral Unit's activation}.
```
Typos

  \item Each player can cast only \textbf{one Arcane Artillery per combat Round}.
  \item \textit{In alliance mode, allies can cast Arcane Artillery on allied Unit's activation.}
\end{itemize}

\end{multicols}

\newpage

>   \begin{minipage}{0.4\paperwidth}

Reviewer: In general, I'd discourage using minipages for this. IMO, tikzpicture works much better, because it's easier to move things around and keep pictures to scale. On that note, `\linewidth` works better, because the former is pinned to the available space, and the latter to the width of the whole page, which in this case makes little sense. But, I guess, in this scenario we can leave it as it is.

Author: Sure. I was lazily copy pasting from the Titans Stronghold scenario since it had a similar amount of maps, but I can use tikzpicture next time

% !TeX spellcheck = en_US

> \addscenariosection{1}{Clash Scenario}{Arcane Artillery}{\images/berserk.png}

Reviewer: Berserk is already used, please pick something else from here: https://github.com/Heegu-sama/Homm3BG/tree/main/assets/spells

\vspace*{\fill}

\subsection*{\MakeUppercase{Defeat Conditions}}
At the end of the \nth{12} Round, if there is no winner, all players lose the Scenario.

\subsection*{\MakeUppercase{Timed Events}}

\textbf{\nth{6} Round:}
\begin{itemize}
  \item Remove all Black cubes from every Tile.
\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}

Before the start of this Scenario:

\begin{itemize}
  \item If playing Alliance mode, allies and oppoents choose starting Tiles alternately.
\end{itemize}

During this Scenario:

\begin{itemize}
  \item Heroes cannot enter starting Tiles other than their own.
  \item A player may not use the Diplomacy card to skip Combat on the Dragon Utopia Field.
  \item Arcane Spikes are represented by cubes not used by any player's Faction.
\end{itemize}

\vspace*{\fill}

\columnbreak

\begin{itemize}
    \item Obelisks may be captured. The first time an Obelisk is captured, gain one Arcane Spike.
    \item At the start of each player's turn, that player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of captured Obelisks they have.
\end{itemize}

\vspace{1em}

> \includegraphics[width=\linewidth, keepaspectratio]{\art/griffin.png}

Reviewer: You already used the griffin, please pick something else. Here's some inspiration: https://github.com/Heegu-sama/Homm3BG/tree/main/assets/art

\end{multicols}

\vspace{1em}

\hommtable[]{15}{
  \begin{tabularx}{\linewidth}{X}
  \darkcell{\raisebox{-.2\height}{} \textbf{Activation Ability: Arcane Artillery}}\\

>   \lightcell[4.8]{\raisebox{-.25\height}{} \includegraphics[height=10px]{\layout/listdot.png} Deal 1 \svg{damage-table} to target Unit by placing an Arcane Spike on it. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery ignores the Azure Dragon's special ability. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery does not count toward combat Round spell limit. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery is considered to be a spell of every school of magic. \\ \includegraphics[height=10px]{\layout/listdot.png} Players may cast Arcane Artillery when an opponent is in combat with neutral Units, on the neutral Units turn. \\ \includegraphics[height=10px]{\layout/listdot.png} Each player may only cast Arcane Artillery once per combat Round. \\ \includegraphics[height=10px]{\layout/listdot.png} If playing Alliance mode, allies may cast Arcane Artillery on allied player's Unit activation.} \\

Reviewer: I suggest making individual cells for every rule, like here:
![image](https://github.com/user-attachments/assets/bf023020-fa87-425d-93f0-beff7c8f4f0e)

Reviewer: Also, please split every row to a separate line.
